src/test/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshHeadersTest.java

```diff
@@ -34,7 +33,7 @@ void createSendHeaders() {
             "Mex-FileName",
             "Mex-MessageType");
 
-        BasicHeader mexToHeader = Arrays.stream(meshHeaders.createSendHeaders(meshRecipient))
+        BasicHeader mexToHeader = Arrays.stream(meshHeaders.createSendHeaders(meshRecipient, WorkflowId.REGISTRATION))
```

Comment:
Should we also test the other headers with dynamic values here?
* Mex-From
* Mex-WorkflowID

Reply:
We should probably